frontend/src/components/Pages/InspectionPage/ScheduleMissionDialogs.tsx

```diff
@@ -78,29 +74,13 @@ export const ScheduleMissionDialog = (props: IProps): JSX.Element => {
     }, [enabledRobots, installationCode])
 
     const onSelectedRobot = (selectedRobot: Robot) => {
-        if (!robotOptions) return
-
-        setSelectedRobot(selectedRobot)
+        if (robotOptions) setSelectedRobot(selectedRobot)
     }
 
-    const onScheduleButtonPress = () => {
+    const onScheduleButtonPress = (missions: CondensedMissionDefinition[]) => () => {
```

Comment:
The error was actually made in a rebase further down in the file. It should be:
![image](https://github.com/equinor/flotilla/assets/20266900/ede0fbd7-effc-4c5c-9373-9a3d7deb5553)
instead of 
![image](https://github.com/equinor/flotilla/assets/20266900/be4090ce-5e82-4f47-ad5f-b2d60337a282)